app/scripts/modules/core/pipeline/config/actions/history/showHistory.controller.js

```diff
@@ -3,20 +3,21 @@
 const angular = require('angular');
 
 import {PIPELINE_CONFIG_SERVICE} from 'core/pipeline/config/services/pipelineConfig.service';
+import {JSON_UTILITY_SERVICE} from 'core/utils/json/json.utility.service';
 
 require('./showHistory.less');
 
 module.exports = angular
   .module('spinnaker.core.pipeline.config.actions.history.controller', [
     PIPELINE_CONFIG_SERVICE,
     require('core/utils/jQuery'),
-    require('./jsonDiff.service'),
+    JSON_UTILITY_SERVICE,
     require('./diffSummary.component'),
     require('./diffView.component'),
   ])
-  .controller('ShowHistoryCtrl', function ($window, pipelineConfigService, jsonDiffService,
+  .controller('ShowHistoryCtrl', function ($window, pipelineConfigService, jsonUtilityService,
                                            pipelineConfigId, currentConfig,
-                                           $uibModalInstance, $, $filter) {
```

Comment:
how the heck did this even work before?  i thought we removed the jquery service last fall.